api/task.go

```diff
@@ -576,7 +583,7 @@ type TaskReport struct {
 	Total     int         `json:"total"`
 	Completed int         `json:"completed"`
 	Activity  []string    `json:"activity"`
-	Result    interface{} `json:"result,omitempty"`
+	Result    interface{} `json:"result,omitempty" swaggertype:"object"`
```

Comment:
hint: necessary to specifically set the swaggertype for the `interface{}` field to prevent a segfault when generating docs